dts/bindings/dac/ti,dac-x0501.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 Google, LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+include: base.yaml
+
+description: TI DACx0501 12 to 16 bit DAC
+
+compatible: "ti,dac-x0501"
+
+properties:
+  ref-source:
+    type: string
+    required: true
+    enum:
+      - "internal"
+      - "external"
+    description: |
+      DAC voltage reference select: either internal or external
+
+  voltage_mult:
+    type: string
+    enum:
+      - "mul2"
+      - "mul1"
+      - "div2"
+    default: "mul1"
+    description: |
+      This setting can be used to control the output voltage range within the supported bit
+      resolution. mul2 will double the output range but lower the resolutiuon, while div2 will
+      lower the range but double the resolution. The default is arbitrarily set to the middle
+      point (mul1) which should fit most applications (if they care about it)
```

Comment:
Please change the last sentence to "The default is set to the middle setting mul1 to fit most use cases."

Reply:
Done

Reply:
Removed the default and made it required instead per @martinjaeger suggestion